library/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.android.library'
```

Comment:
Please change this and package to `com.android.gpstest.library`

Reply:
> Please change this and package/folder for classes to `com.android.gpstest.library`

Ok, thanks.